src/pages/resultsView/ResultsViewPage.tsx

```diff
@@ -542,34 +543,49 @@ export default class ResultsViewPage extends React.Component<
         return isRoutedTo || (!isExcludedInList && !isExcluded);
     }
 
-    @computed get quickOQLSubmitButtion() {
-        return (
-            <>
-                <button
-                    className={'btn btn-primary btn-sm'}
-                    style={{ marginLeft: 10 }}
-                    onClick={this.handleQuickOQLSubmission}
-                >
-                    Submit Query
-                </button>
-                &nbsp;
-                <button
-                    className={'btn btn-link btn-sm'}
-                    onClick={this.toggleOQLEditor}
-                >
-                    Cancel
-                </button>
-            </>
-        );
+    @computed get quickOQLSubmitButton() {
```

Comment:
i think submitting store (instead of just changing the gene_list param) will cause us to lose settings kept in url?

Reply:
good catch. fixed this by passing an extra arg to make the URL not clear here: https://github.com/cBioPortal/cbioportal-frontend/pull/3231/files#diff-3cdd9cf2b536b6b0a606ab1d87e2d04aR578